fhir-examples/src/main/resources/json/ibm/complete-absent/AllergyIntolerance-4.json

```diff
@@ -376,7 +376,7 @@
             ]
         }
     },
-    "_onsetDateTime": {
```

Comment:
What the ExamplesGenerator is *supposed* to be doing is creating a variant of each resource type with each choice data type.  AllergyIntolerance.onsetDateTime is a choice of 5 different types, and so I'd expect there to be 5 instances, one for each choice type:

onsetDateTime |   |   | dateTime |  
-- | -- | -- | -- | --
onsetAge |   |   | Age |  
onsetPeriod |   |   | Period |  
onsetRange |   |   | Range |  
onsetString |   |   | string

Reply:
it was accounting for duplicates in each type... meaning mock has duplicate in absent, minimal and they were deduping each other.  I have now accounted for it.